### PR TITLE
Expose "setOffline" method

### LINKED ISSUE #81: Expose setOffline
## Summary
Expose `setOffline` via the flutter plugin

Both objective-c and Kotlin counterparts offer a method 'setOffline'.

```
/**
     * Sets offline. If offline is true, then the SDK will not upload events to Amplitude servers;
     * however, it will still log events.
     *
     * @param offline whether or not the SDK should be offline
     * @return the AmplitudeClient
     */
    public AmplitudeClient setOffline(boolean offline) {
        this.offline = offline;

        // Try to update to the server once offline mode is disabled.
        if (!offline) {
            uploadEvents();
        }

        return this;
    } 
``` 

```
- (void)setOffline:(BOOL)offline {
    _offline = offline;

    if (!_offline) {
        [self uploadEvents];
    }
}
```

## Motivations

Prevent uploading events to Amplitude server in special conditions.

### PR DESCRIPTION
**Pull request by nishiths23**
Expose the "setOffline" method, already available on the Android/iOS Amplitude SDKs.

Closes the following feature request: https://github.com/amplitude/Amplitude-Flutter/issues/81

### CODE REVIEW

**Comment by nishiths23** [comment]
@yuhao900914 tagging maintainers for review

File: lib/amplitude_web.dart [hunk 1]
```diff
@@ -177,6 +178,8 @@ class AmplitudeFlutterPlugin {
           bool updateServerUrl = args['updateServerUrl'];
           return amplitude.setServerZone(serverZone, updateServerUrl);
         }
+      case "setOffline":
+        return false;
```
Comment (justin-fiedler) [inline lib/amplitude_web.dart]: Please use standard code style
```
case "setOffline":
  {
    return false;
  }
```

**Review by justin-fiedler — COMMENTED** [review]
Thanks @nishiths23 for the feature. Overall looks good, see nit about code style.

**Review by yuhao900914 — APPROVED** [review]

File: lib/amplitude_web.dart (hunk 1 above)
Comment (nishiths23) [reply lib/amplitude_web.dart]: Updated

**Pull request merged**